### PR TITLE
megatools: package at 1.10.3

### PR DESCRIPTION
**Pull request by mmatongo**
---
## Description of package
Megatools is a collection of programs for accessing Mega.nz service from a command line of your desktop or server.

--

## Installed manifest of package

```
/var/db/kiss/installed/megatools/version
/var/db/kiss/installed/megatools/sources
/var/db/kiss/installed/megatools/manifest
/var/db/kiss/installed/megatools/depends
/var/db/kiss/installed/megatools/checksums
/var/db/kiss/installed/megatools/build
/var/db/kiss/installed/megatools/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man7/megatools.7
/usr/share/man/man7/
/usr/share/man/man5/megarc.5
/usr/share/man/man5/
/usr/share/man/man1/megarm.1
/usr/share/man/man1/megareg.1
/usr/share/man/man1/megaput.1
/usr/share/man/man1/megamkdir.1
/usr/share/man/man1/megals.1
/usr/share/man/man1/megaget.1
/usr/share/man/man1/megadl.1
/usr/share/man/man1/megadf.1
/usr/share/man/man1/megacopy.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/doc/megatools/
/usr/share/doc/
/usr/share/
/usr/bin/megarm
/usr/bin/megareg
/usr/bin/megaput
/usr/bin/megamkdir
/usr/bin/megals
/usr/bin/megaget
/usr/bin/megadl
/usr/bin/megadf
/usr/bin/megacopy
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/megatools/build [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --disable-docs
+
+make
+
+./install-sh -c -d "$1/usr/bin"
+./install-sh -c -d "$1/usr/share/doc/megatools"
```
Comment (dylanaraps) [inline community/megatools/build:10]: This directory is created but nothing is ever copied to it?

File: community/megatools/build [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+./install-sh -c -m 644 docs/megaget.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megals.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megamkdir.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megaput.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megareg.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megarm.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megacopy.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megadl.1 "$1/usr/share/man/man1"
+./install-sh -c -m 644 docs/megarc.5 "$1/usr/share/man/man5"
+./install-sh -c -m 644 docs/megatools.7 "$1/usr/share/man/man7"
```
Comment (dylanaraps) [inline community/megatools/build:33]: No `make install` or some kinda issue with it?

Comment (dylanaraps) [reply community/megatools/build:33]: I see. This is to avoid docs but keep manual pages.

Comment (dylanaraps) [reply community/megatools/build:33]:
```sh
#!/bin/sh -e

export A2X=/bin/true

./configure \
    --prefix=/usr

make
make DESTDIR="$1" install
```

This works for me. Installs manual pages and documentation.

Comment (mmatongo) [reply community/megatools/build:33]: Yeah, it was for that.
I'll update the build file

File: community/megatools/build (hunk 1 above)
Comment (mmatongo) [reply community/megatools/build:10]: I forgot to remove that when i added the `--disable-docs` flag

File: community/megatools/build (hunk 2 above)
Comment (mmatongo) [reply community/megatools/build:33]: I'd like to maintain not having the documentation installed.

Comment (mmatongo) [reply community/megatools/build:33]: They aren't all that necessary

Comment (dylanaraps) [reply community/megatools/build:33]: Just delete them :+1:

Comment (dylanaraps) [reply community/megatools/build:33]:
```sh
rm -rf "$1/usr/share/doc"
```

Comment (mmatongo) [reply community/megatools/build:33]: The man pages are just fine

Comment (dylanaraps) [reply community/megatools/build:33]: Yeah

**Pull request merged**